charts/talks/templates/ingress.yaml

```diff
@@ -0,0 +1,37 @@
+{{- $serviceName := include "fullname" . -}}
+{{- $servicePort := .Values.service.externalPort -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    {{- range $key, $value := .Values.ingress.annotations }}
+      {{ $key }}: {{ $value | quote }}
+    {{- end }}
+    {{- if .Values.ingress.globalStaticIpName }}
+      kubernetes.io/ingress.global-static-ip-name: {{ .Values.ingress.globalStaticIpName }}
+    {{- end }}
+spec:
+  rules:
+    {{- range $host := .Values.ingress.hosts }}
```

Comment:
hmm, in case we need to support multiple dns entries... I will add the required tag